draft-bbs-signatures.md

```diff
@@ -411,13 +436,15 @@ Procedure:
 
 4. if KeyValidate(pub_key) is INVALID
 
-5. B = P1 + H0 * s + H_1 * msg_1 + ... + H_L * msg_L
+5. sig_dst = OS2IP(HASH(PK || L || H_s || H_d || H_1 || ... || H_L || Ciphersuite_ID || CipherInfo)) mod q
```

Comment:
Considering we are talking about having the parmeters suitable implicitly converted would the same make sense to the ouput of the HASH function e.g always returns the form of OS2IP?

Reply:
Thank you! That's a good point. I opened PR #101 which is towards this direction and if adopted will replace the OS2IP(HASH(...)) here.